command/ssh/certificate.go

```diff
@@ -254,6 +261,28 @@ func certificateAction(ctx *cli.Context) error {
 		if err != nil {
 			return err
 		}
+
+		// All host identity certs need a URI SAN to work with our ssh API.
+		if isHost {
```

Comment:
There's a switch on line 195 that we can use to throw an error if `--host-id` is passed without `--host`

Reply:
done